sql/moz-fx-data-shared-prod/mozilla_vpn_derived/funnel_product_page_to_subscribed_v1/query.sql

```diff
@@ -127,13 +127,13 @@ flows AS (
       AND user_id IS NOT NULL
     ) AS pay_setup_engage_with_uid,
     -- new fxa after entering the email
-    LOGICAL_OR(event_type = "fxa_pay_setup - 3ds_complete") AS pay_setup_complete,
+    LOGICAL_OR(event_type = "fxa_pay_setup - success") AS pay_setup_complete,
```

Comment:
In all these conditions we need to continue to check for the historical event name, otherwise we'll have to time this deploy exactly when the FxA changes get deployed and we couldn't re-run this ETL for any historical dates.
```suggestion
    LOGICAL_OR(event_type IN ("fxa_pay_setup - 3ds_complete", "fxa_pay_setup - success")) AS pay_setup_complete,
```

Reply:
Good point. Thanks!